service/routes.py

```diff
@@ -89,6 +90,126 @@ def create_wishlists():
         status.HTTP_201_CREATED
     )
 
+######################################################################
+# RETRIEVE A WISHLIST
+######################################################################
+@app.route("/wishlists/<int:wishlist_id>", methods=["GET"])
+def get_wishlist(wishlist_id):
+    """
+    Retrieve a single wishlist
+    This endpoint will return a wishlist based on it"s id
+    """
+    app.logger.info("Request for wishlist with id: %s", wishlist_id)
+    wishlist = Wishlist.find_by_id(wishlist_id)
+    if not wishlist:
+        return(
+            f"Wishlist with id {wishlist_id} was not found",
+            status.HTTP_404_NOT_FOUND
+        )
+    return make_response(jsonify(wishlist.serialize()), status.HTTP_200_OK)
+
+
+######################################################################
+# ADD A PRODUCT IN A WISHLIST
+######################################################################
+@app.route("/wishlists/<int:wishlist_id>/products", methods=["POST"])
```

Comment:
I'm wondering if this should be a `PUT` since you're updating an existing wishlist.